darwin/importer/formats/nifti.py

```diff
@@ -513,16 +522,22 @@ def correct_nifti_header_if_necessary(img_nii):
 def process_nifti(
     input_data: nib.nifti1.Nifti1Image,
     ornt: Optional[List[List[float]]] = [[0.0, -1.0], [1.0, -1.0], [2.0, -1.0]],
+    remote_file_path: Path = Path("/"),
+    remote_files_that_require_legacy_scaling: Dict[Path, Dict[str, Any]] = {},
 ) -> Tuple[np.ndarray, Tuple[float]]:
     """
     Converts a nifti object of any orientation to the passed ornt orientation.
     The default ornt is LPI.
 
     Args:
         input_data: nibabel nifti object.
-        ornt: (n,2) orientation array. It defines a transformation from RAS.
+        ornt: (n,2) orientation array. It defines a transformation to LPI
```

Comment:
`nibabel` and `ITK` disagree on what orientation is represented by `LPI`, `RAS`, etc. We process `NifTI` annotations exclusively with `nibabel`, which represents `LPI` as:

`[[0.0, -1.0], [1.0, -1.0], [2.0, -1.0]]`